test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -336,9 +337,10 @@ var _ = SIGDescribe("CustomResourcePublishOpenAPI", func() {
 			e2elog.Failf("%v", err)
 		}
 
+		// TODO(jpbetz): Use json patch to update the name? This doesn't work.
```

Comment:
not sure what "This doesn't work." refers to. This will rename "v3" to "v4" (and fail if something in the setup changes so versions[1] is no longer v3:

```
[
  {"op":"test","path":"/spec/versions/1/name","value":"v3"}, 
  {"op":"replace","path":"/spec/versions/1/name","value":"v4"}
]
```

Reply:
TODO was stale, I've removed. Including the `test` check is a good improvement. I've added it.

Reply:
still seeing the todo?

Reply:
Oops. Gone now.